kevinhartmann23/fitlit
# Feature/responsiveness

#### What does  this PR do?

This PR begins adding media queries for responsiveness.

#### Where should the reviewer start?

Familiarize yourself with the media queries added to styles.css

#### Is this a bug fix or a feature?

Feature

#### How should this be manually tested?

In browser, change the view between devices to ensure breakpoints are functioning properly.
